Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -168,6 +171,20 @@ extension _EasyHandle {
         let protocols = (CFURLSessionProtocolHTTP | CFURLSessionProtocolHTTPS)
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
+        #if os(Android)
+            // See https://curl.haxx.se/docs/sslcerts.html
+            // For SSL to work you need "cacert.pem" to be accessable
+            // at the path pointed to by the URLSessionCAInfo env var.
```

Comment:
Roger, not much I can do about the last one. There is no stdout on Android. Can you check https://github.com/johnno1962b/swift-corelibs-foundation/commit/d3805cb9e361b7595e24980b7a2fc4810efcaa18 please and I’ll raise a new PR tomorrow. Thanks again!